lndr-cli/test/Spec.hs

```diff
@@ -345,7 +345,8 @@ basicSettlementTest = do
     gottenTxHash <- getTxHash testUrl creditHash
     assertEqual "successful txHash retrieval" txHash (addHexPrefix gottenTxHash)
 
-    (dbCredits, blockchainCredits, _) <- getUnsubmitted testUrl
+    s@(dbCredits, blockchainCredits, _) <- getUnsubmitted testUrl
+    print s
```

Comment:
I know this is still a work-in-progress -- just pointing out this print statement so we don't forget to remove it.

Reply:
Yep, thanks.